deploy.sh

```diff
@@ -10,13 +10,18 @@ export TERRAFORM_ROOT_OPERATIONS="$BITOPS_OPSREPO_ENVIRONMENT_DIR"
 export BITOPS_SCHEMA_ENV_FILE="$TERRAFORM_ROOT_OPERATIONS/ENV_FILE"
 export SCRIPTS_DIR="$TERRAFORM_ROOT_SCRIPTS/scripts"
 
+if [ -n "$SKIP_DEPLOY_TERRAFORM" ]; then
```

Comment:
If ENV var is set like `SKIP_DEPLOY_TERRAFORM=0` or `SKIP_DEPLOY_TERRAFORM=` it'll still skip the deployment anyway.

```suggestion
if [ "${SKIP_DEPLOY_TERRAFORM}" == "true" ]; then
```
Docs are referring to `true` value for this setting to work properly.
https://github.com/bitops-plugins/terraform/blob/main/README.md#additional-environment-variable-configuration

Let's use that flag instead.